app/models/user.rb

```diff
@@ -607,6 +607,15 @@ def self.whitelist_for_changes_display
     )
   end
 
+  def skip_session_limitable?
+    # allow multiple concurrect sessions in development
+    if Rails.env.development?
+      true
+    else
+      false
+    end
+  end
+
```

Comment:
unrelated to this PR, but figured we could disable this in development

Reply:
I wonder if we should move that to an ENV with a default of false.  Something like:
```suggestion
  def skip_session_limitable?
    ENV.fetch('SESSION_LIMITABLE', false)
  end

```
I'm worried that if we blanket disable things in development, we don't thoroughly test them while we're building.

Reply:
Good point! Will update. Calling it `SKIP_SESSION_LIMITABLE` since that's what this method is asking.

Reply:
Booleans with dotenv are always annoying, the best way I could think to write it is `ENV.fetch('SKIP_SESSION_LIMITABLE') == 'true'`. Because what you wrote would return `"true"` not `true`.